internal/credentials/util_test.go

```diff
@@ -58,3 +58,29 @@ func (s) TestAppendH2ToNextProtos(t *testing.T) {
 		})
 	}
 }
+
+func (s) TestRemoveServiceNameFromJwtURI(t *testing.T) {
+	tests := []struct {
+		name string
+		uri  string
+		want string
+	}{
+		{
+			name: "invalid URI",
```

Comment:
Does this case want an error?  Test the error?

Reply:
Done.